docker-compose.yaml

```diff
@@ -52,6 +56,30 @@ services:
     environment:
       BASE_URL: "http://backend:3000/api/v3"
       FACILITY: "SAMPLE-SITE"
+  local-rabbitmq:
+    hostname: local-rabbitmq
+    build: 'rabbitmq/.'
```

Comment:
this seems a little overcomplitacted. Can't we use the default rabbitmq user and password, with the default health check? also, do we need the rabbitmq console? if not I don't think we need to use traefik for this

Reply:
Not sure if the guest account is enough for creating a new queue? Though I haven't tried it either..

Reply:
https://hub.docker.com/_/rabbitmq here it says what the defaults are

Reply:
Apparently, the guest account can only connect from "localhost", all other connections are blocked.

Reply:
As reference (from https://www.rabbitmq.com/docs/access-control):
![image](https://github.com/SciCatProject/scicatlive/assets/32138106/690cebec-14c3-4110-b94c-1b52a15f871d)

Reply:
Instead of pinging the website, one could use `rabbitmq-diagnostics -q ping` with return code 0 from within the container to check RabbitMQ's healthiness.

Reply:
I have just tried, with the `guest` user and it works fine. The problem is though that the readiness check is not enough and we need to wait for a health check (as you are doing). I wonder if we can streamline that.

for the console, if you use it, let's keep it!

Reply:
I am fine with the health check in the rabbitmq container but I'd like to avoid installing curl in the container and use the official one

Reply:
I like this `rabbitmq-diagnostics -q ping`!

Reply:
Have you tested the `guest` account work from other containers (mainly the scicat backend)?

Reply:
Nevermind, I think in recent versions of the image, the default user can be redefined as well

Reply:
yes. There is no need to redefine anything though, it just works